### ISSUE TITLE
when I run   ./waf --run "rl-tcp",these errors accurred.

### ISSUE THREAD

**Issue by LiYufengzz** [hust-diangroup/ns3-ai#39]
In file included from ../scratch/rl-tcp/tcp-rl-env.cc:2:
../scratch/rl-tcp/tcp-rl-env.h:6:10: fatal error: 'ns3/ns3-ai-module.h' file not found
#include "ns3/ns3-ai-module.h"
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.

In file included from ../scratch/rl-tcp/tcp-rl.cc:1:
In file included from ../scratch/rl-tcp/tcp-rl.h:6:
../scratch/rl-tcp/tcp-rl-env.h:6:10: fatal error: 'ns3/ns3-ai-module.h' file not found
#include "ns3/ns3-ai-module.h"
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.

../scratch/rl-tcp/sim.cc:19:10: fatal error: 'ns3/ns3-ai-module.h' file not found
#include "ns3/ns3-ai-module.h"
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.

Could you tell me why?thanks a lot

**Comment by Mauriyin** [comment]
I think you need to run `./waf configure ` again and then try to compile the new example.

**Comment by LiYufengzz** [comment]
yes,I get errors when I compile ns3 use"./waf "but I don't know the reason.Anyway,thanks for the reply



---Original---
From: ***@***.***&gt;
Date: Thu, Jun 24, 2021 14:05 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hust-diangroup/ns3-ai] when I run   ./waf --run "rl-tcp",these errors accurred. (#39)




 
I think you need to run ./waf configure  again and then try to compile the new example.

**Comment by Mauriyin** [comment]
where did you put the ns3-ai model folder? If it is under contrib/, it should be able to link.

The error means it couldn't find ns3-ai, so I suggest you configure again:
```
./waf configure
```
Then you can check if there is ns3-ai in the log.

**Comment by LiYufengzz** [comment]
Ok,I will have a try

**Comment by LiYufengzz** [comment]
Hi,I have a question. **./waf --run "rl-tcp"** this step will cost a lot of time,right? It has stuck at here for a long time.

**Comment by Mauriyin** [comment]
you can use this [script](https://github.com/hust-diangroup/ns3-ai/blob/master/example/rl-tcp/run_rl_tcp.py) to run:
```
python3 run_rl_tcp.py
```

**Comment by LiYufengzz** [comment]
I use this: python3 run_rl_tcp.py 
build
cleaning
Is the result right?

**Comment by LiYufengzz** [comment]
I can run the run_rl_tcp.py correctly,but the output figures are empty.I want to ask what the problem is.Thanks a lot.

**Comment by igs3000** [comment]
> I can run the run_rl_tcp.py correctly,but the output figures are empty.I want to ask what the problem is.Thanks a lot.

Same problem - I am also getting blank graphs in[ my test also](https://github.com/hust-diangroup/ns3-ai/issues/50)
